ramls/subscribe-callback.raml

```diff
@@ -0,0 +1,34 @@
+#%RAML 1.0
+title: Circulation subscribe callbacks
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Circulation Subscriber Callback Endpoints
+    content: <b>API to handle events</b>
+types:
+  errors: !include raml-util/schemas/errors.schema
+
+/circulation/subscribers/:
+  /fee-fine-with-loan-closed:
+    displayName: Subscriber callback endoint to handle fee/fine record with loan closed event
+    post:
+      description: Handle fee/fine record with loan closed event
+      body:
+        application/json:
+          type: !include subscribers/fee-fine-with-loan-closed.json
+          example: !include examples/fee-fine-with-loan-closed.json
+      responses:
+        204:
+          description: "Event handled successfully"
+        422:
+          description: "Unable to process event"
+          body:
+            application/json:
+              type: errors
+        500:
+          description: "Internal server error"
+          body:
+            text/plain:
+              example: "Internal server error"
```

Comment:
What does pub-sub do with the response from an event notification?

Reply:
It seems it has an internal journal, where it tracks rejected events. I guess it also decides whether the event successfully processed and can be removed from the queue, based on response status code.

Reply:
> It seems it has an internal journal, where it tracks rejected events. I guess it also decides whether the event successfully processed and can be removed from the queue, based on response status code.

Is that reaction documented? 

For example, is it stated that a pub-sub message is only removed from the queue for a subscriber when it has been processed successfully?

I think this is important, because if failed messages are re-delivered, then responding with a 422 or 500 could clog up the queue.

Reply:
No, it is not documented.

Reply:
Please can you reach out to the FoliJet team and ask them to describe (and document) this behaviour so that subscribers know what response they should be issuing to pub-sub

If needed, please raise a JIRA describing what needs documenting (feel free to say that I asked you to). If you are uncomfortable doing that, please let me know and I will raise it.

Reply:
https://issues.folio.org/browse/MODPUBSUB-91 created.

As for now, the only status treated as success is 204. All other statuses will take no effect.

Reply:
Thank you for confirming this